### DISCUSSION TITLE
Codespaces x GitHub Mobile?

### DISCUSSION THREAD

**Discussion by alexavil** [community/community#16753] [Codespaces] [answered]
With GitHub Mobile installed, trying to create a Codespace (from the web interface) always throws an error, since it attempts to redirect you to the app.

![Screenshot_2022-05-17-09-09-51-554_com.github.android.jpg](https://user-images.githubusercontent.com/32246131/168733660-11edf750-8fff-459e-8da4-248c8b764d61.jpg)

In fact, GitHub Mobile is missing a lot of functionality related to Codespaces, so I wonder if at least basic features could be added?

**Comment by dmilburn** [comment, marked as answer]
Hi, thanks so much for your feedback. I've added an internal issue to track this work. Thank you!

**Reply by hifocus** [reply]
github.com on mobile keep redirecting me to the mobile app. But if the page I am accessing does not exist in the app, the app shows an error and the browser page wouldn't work as it will keep redirecting.

Please consider adding a button for the redirect to app. It never should be a wildcard thing.

**Reply by tsarthur** [reply]
Yes, I like it too!

**Reply by djbrown** [reply]
@dmilburn is the issue publicly viewable? If Not: did it get tracktion or are there any Updates to it? Kind regards, daniel

**Reply by DamianEstey** [reply]
You can access vscode from your device. In fact I've done it today. I will want you, you do not want to do this as its a pain in the butt.

**Reply by djbrown** [reply]
@DamianEstey how did you do it, did you follow a specific guide?
And what is painful, setting it up or using it for development?
I would prefer to eventually use GitHub Codespace (remote vscode) but maybe you can give a first good step to achieve that.

**Reply by DamianEstey** [reply]
What I did was download Bing but for separate reasons. When I go to the GitHub account on the browser (not application); I click on codespaces. When it asks me to redirect, I use Bing instead of Chrome and it worked for me.

**Reply by DamianEstey** [reply]
It'll of course ask you to redirect twice, thing is the experience sucks simply because of how much wiggle room you actually get for the actual code editing part; you cannot zen on mobile it seems.

**Comment by DEAD1nsane** [comment]
@alexavil for it to not open the app, go to github app info, by tapping the icon in recent apps or holding down on the app icon in app drawer or where ever you got it, tap set as default and uncheck the only option.
![Screenshot_20240202_003022_One UI Home](https://github.com/community/community/assets/133211494/d03c1910-f788-45da-bff4-fae46cdd56ac)
![Screenshot_20240202_003033_Settings](https://github.com/community/community/assets/133211494/706983e4-280a-4cdd-9d27-8057d2aee99a)
![Screenshot_20240202_003040_Settings](https://github.com/community/community/assets/133211494/12e9f992-0064-4ab0-b074-8c41ba3f4540)
![Screenshot_20240202_003347_Chrome](https://github.com/community/community/assets/133211494/f17e111e-0050-4af7-9bb4-b208b28ff3a2)
